### PR TITLE
Recaptcha integration

### PR DESCRIPTION
**Pull request by znarf**

### CODE REVIEW

**Comment by xdamman** [comment]
Is there an issue linked to it?
What's the behavior of this?

**Comment by znarf** [comment]
@xdamman https://github.com/opencollective/opencollective/issues/1457

File: migrations/201811160000-add-data-to-orders.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        console.log('>>> done');
+      });
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface
+      .removeColumn('Orders', 'data')
+      .then(() => queryInterface.removeColumn('OrderHistories', 'data'))
+      .then(() => {
+        console.log('>>> done');
```
Comment (marcogbarcellos) [inline migrations/201811160000-add-data-to-orders.js]: not a big deal but we could remove this console log later!

Comment (znarf) [reply migrations/201811160000-add-data-to-orders.js]: @marcogbarcellos It's a migration

Comment (marcogbarcellos) [reply migrations/201811160000-add-data-to-orders.js]: you could then improve the message on the console log as when you run the sequelize `db:migrate:undo` it'll run all files and will perhaps not be that clear it's regarding this file.

Comment (znarf) [reply migrations/201811160000-add-data-to-orders.js]: I just did this migration like multiple before. If these `'>>> done'` messages are a problem, then we should remove all of them.

**Pull request merged**